src/foam/core/stdlib.js

```diff
@@ -1000,8 +1000,8 @@ foam.LIB({
       return date.toDateString().substring(4);
     },
     /** formats a date as MMM dd, YYYY
-     *  Use options includeTime and timeFirst to include a time
-     *  string hh:mm:ss at the start of end of the date string
+     *  Use the timeFirst parameter to include a time string
```

Comment:
*  Use the timeFirst parameter to include a time string.   If the parameter is not provided, no time information will be encoded in the final format.  When provided and true, time is prefixed, else postfixed.

Reply:
thanks, updated @jlhughes